package.json

```diff
@@ -105,12 +106,12 @@
   },
   "devDependencies": {
     "@microsoft/eslint-plugin-sdl": "^0.1.5",
-    "@types/chai": "4.0.6",
+    "@types/chai": "^4.0.6",
     "@types/fuse": "2.5.2",
     "@types/highlight.js": "9.12.2",
     "@types/jquery": "3.3.29",
     "@types/marked": "0.3.0",
-    "@types/mocha": "2.2.44",
+    "@types/mocha": "^2.2.44",
```

Comment:
This is risky unless we check in package-lock.json. This introduces uncertainty around what we develop against vs. what we ship with.

Reply:
Is now a good time to include package-lock.json?

Reply:
one thing at a time! but i will revert this, not sure how it even happened